### ISSUE TITLE
[Issue]: 

### ISSUE THREAD

**Issue by alexmilanionline** [jellyfin/jellyfin#8954]
### Please describe your bug

Today I tried accessing Jellyfin will not load localhost where the actual V10.8.8. USing Windows 10 Pro OS BUILD: 19045.2364 open when to Jellyfin tray app open Jellyfing welcoming jellyfin logo comes up and get stuck can anyone help I have tried everything??? Also when I tried to access from another computer I get this error message??? http://192.168.138.169:8096/http://192.168.138.169:8920/web/index.html#!/home.html/web/index.html

And I also get this message? Can someone help and resolve this please?
The page isn’t redirecting properly

Firefox has detected that the server is redirecting the request for this address in a way that will never complete.

This problem can sometimes be caused by disabling or refusing to accept cookies.

### Jellyfin Version

10.8.0

### Environment

```markdown
Today I tried accessing Jellyfin will not load localhost where the actual V10.8.8. USing Windows 10 Pro OS BUILD: 19045.2364 open when to Jellyfin tray app open Jellyfing welcoming jellyfin logo comes up and get stuck can anyone help I have tried everything??? Also when I tried to access from another computer I get this error message??? http://192.168.138.169:8096/http://192.168.138.169:8920/web/index.html#!/home.html/web/index.html

And I also get this message? Can someone help and resolve this please?
The page isn’t redirecting properly

Firefox has detected that the server is redirecting the request for this address in a way that will never complete.

This problem can sometimes be caused by disabling or refusing to accept cookies.
```

### Please attach any screenshots here

Today I tried accessing Jellyfin will not load localhost where the actual V10.8.8. USing Windows 10 Pro OS BUILD: 19045.2364 open when to Jellyfin tray app open Jellyfing welcoming jellyfin logo comes up and get stuck can anyone help I have tried everything??? Also when I tried to access from another computer I get this error message??? http://192.168.138.169:8096/http://192.168.138.169:8920/web/index.html#!/home.html/web/index.html

And I also get this message? Can someone help and resolve this please?
The page isn’t redirecting properly

Firefox has detected that the server is redirecting the request for this address in a way that will never complete.

This problem can sometimes be caused by disabling or refusing to accept cookies.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by richjeffery** [comment]
I recently had that issue too. For me, it was caused by me mis-setting the 'Base URL' in the Network settings, so it was redirecting to a page that didn't exist, and it was redirected back to itself, which caused a loop and a 'the server is redirecting the request for this address in a way that will never complete' message.

It looks like you may have accidentally set the Base URL to `/http://192.168.138.169:8920/web/index.html#!/home.html)` which in turn is causing a bad redirect. Clearing it should get it working again.

As it sounds like you've lost access to the web frontend, you'll need to go into the place you have installed Jellyfin, then in the file `config/network.xml`, clear the stuff between the `<BaseURL></BaseURL>` brackets. Restart Jellyfin and see whether you can access it again.

**Comment by alexmilanionline** [comment]
This is what my issues is with Jellyfin

On 12/25/2022 when I started again jellyfin using Windows 10 when I tried to sign in I got connection failure? I thought It was my hard drive it was a 4TB all sectors checked out ok checked library.db was ok, I had on 275GB space left on my hard drive someone on Jellyfin community told me to get a bigger hard drive I cloned all data to 14 TB hard drive started the machine up same issue connection failure? I was advised to reinstall Jellyfin from scratch so I did in 10.8.8. lasts version, it told me that I had previous jellyfin server installed I have already uninstalled it tried again same thing somehow finally I got pass all that than when I was told to create new libraries in Jellyfin I did that it started to redownload all metadata again I managed to save cache folder, and metadata folder placed them in same directory still no luck  I even went to options.xml ? I am completely puzzled all my media still there, can anyone help maybe there additional files I need to delete from old Jellyfin setups?
